homeassistant/components/solarlog/const.py

```diff
@@ -17,83 +30,202 @@
+        json_key="powerDC",
+        name="power DC",
+        icon="mdi:solar-power",
+        unit_of_measurement=POWER_WATT,
+        device_class=DEVICE_CLASS_POWER,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SolarLogSensorEntityDescription(
+        key="voltage_ac",
+        json_key="voltageAC",
+        name="voltage AC",
+        icon="mdi:flash",
+        unit_of_measurement=ELECTRIC_POTENTIAL_VOLT,
+        device_class=DEVICE_CLASS_VOLTAGE,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SolarLogSensorEntityDescription(
+        key="voltage_dc",
+        json_key="voltageDC",
+        name="voltage DC",
+        icon="mdi:flash",
+        unit_of_measurement=ELECTRIC_POTENTIAL_VOLT,
+        device_class=DEVICE_CLASS_VOLTAGE,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SolarLogSensorEntityDescription(
+        key="yield_day",
+        json_key="yieldDAY",
+        name="yield day",
+        icon="mdi:solar-power",
+        unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        device_class=DEVICE_CLASS_ENERGY,
+        state_class=STATE_CLASS_MEASUREMENT,
+        last_reset=dt.start_of_local_day(),
+    ),
+    SolarLogSensorEntityDescription(
+        key="yield_yesterday",
+        json_key="yieldYESTERDAY",
+        name="yield yesterday",
+        icon="mdi:solar-power",
+        unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        device_class=DEVICE_CLASS_ENERGY,
+    ),
+    SolarLogSensorEntityDescription(
+        key="yield_month",
+        json_key="yieldMONTH",
+        name="yield month",
+        icon="mdi:solar-power",
+        unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        device_class=DEVICE_CLASS_ENERGY,
+    ),
+    SolarLogSensorEntityDescription(
+        key="yield_year",
+        json_key="yieldYEAR",
+        name="yield year",
+        icon="mdi:solar-power",
+        unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        device_class=DEVICE_CLASS_ENERGY,
+    ),
+    SolarLogSensorEntityDescription(
+        key="yield_total",
+        json_key="yieldTOTAL",
+        name="yield total",
+        icon="mdi:solar-power",
+        unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        device_class=DEVICE_CLASS_ENERGY,
+        state_class=STATE_CLASS_MEASUREMENT,
+        last_reset=dt.utc_from_timestamp(0),
+    ),
+    SolarLogSensorEntityDescription(
+        key="consumption_ac",
+        json_key="consumptionAC",
+        name="consumption AC",
+        icon="mdi:power-plug",
+        unit_of_measurement=POWER_WATT,
+        device_class=DEVICE_CLASS_POWER,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SolarLogSensorEntityDescription(
+        key="consumption_day",
+        json_key="consumptionDAY",
+        name="consumption day",
+        icon="mdi:power-plug",
+        unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        device_class=DEVICE_CLASS_ENERGY,
+        state_class=STATE_CLASS_MEASUREMENT,
+        last_reset=dt.start_of_local_day(),
```

Comment:
This is already added to `consumption_total`, any reason to add it to this one as well?

Reply:
Removed